tests/conformance_suite.postman_collection.json

```diff
@@ -3534,6 +3534,72 @@
 				{
 					"name": "Negative Testing",
 					"item": [
+						{
+							"name": "New Folder",
```

Comment:
@brownoxford, you might want to name this properly.

Reply:
fixed, thanks!